solved_exercises/dani/04-functions.py

```diff
@@ -0,0 +1,63 @@
+print("Calculate the factorial of a number (positive integer).")
+def factorial (n):
+    if n > 0:
+        result = n * factorial(n-1)
+    else:
+        result = 1
+    return result
+value = int(input("Please enter a numeric value:\n"))
+print("Factorial of",value,"is",factorial(value))
+
+print("Write a function that flattens a list.")
+def flat (lista):
+	return [val for sublist in lista for val in sublist]
+given_list = [[18.0], [13.8], [16.2], [56.5], [147], [18.2]]
```

Comment:
this method works only if all list items are lists and it only goes one level deep.

Reply:
ok, I wasn't sure if this is expected from "flatten"